README.md

```diff
@@ -303,7 +303,7 @@ There are many [recipes][7] to assist you with deploying Agent integration confi
 
 ## Resources
 
-### datadog_monitor
+### Datadog monitor
```

Comment:
This is documenting the `datadog_monitor` resource (ie: class-like thing you can instantiate in chef) in our chef cookbook. By renaming it this could be confused with Datadog monitors (which are a completely different thing, unfortunately). I like the explicit resource name better as a title here, but if you don't want to mention code bits in the title, then this shoul be something like "Setting up integrations without explicit recipes".